### ISSUE TITLE
[Accessibility]Inspect: There is an additional “”edit node in Inspect when ListView item in edit Status

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#4810]
* .NET Core Version: 
.Net 6.0.100-preview.4.21218.5 from main branch

* Have you experienced this same bug with .NET Framework?: 
Yes

**More info:**
1.	In .NET Framework 4.7.1, it has the same result.
2.	This issue can repro no matter enable virtual mode or disable virtual mode.

**Minimal repro:**
1.	Extract the “noVirtual_Listview_detail.zip” and open it in VS.
[noVirtual_Listview_detail.zip](https://github.com/dotnet/winforms/files/6334696/noVirtual_Listview_detail.zip)
2.	Build and run this application.
3.	Start Inspect and focus on one ListView item(e.g. item1), observe the structure in Inspect.
4.	Make ListView item in edit status and focus it, observe the structure in Inspect.

**Actual behavior:**
After step3, it focus on “item1” edit node in Inspect.
![step3](https://user-images.githubusercontent.com/45864017/115210632-5ebf5900-a131-11eb-85db-02990af2f621.png)
After step4, there is an additional “”edit node in Inspect when ListView item in edit Status.
![step4](https://user-images.githubusercontent.com/45864017/115210652-62eb7680-a131-11eb-881e-f3e85ae0c046.png)

**Expected behavior:**
After step4, it should focus on “item1” edit node, and no additional “”edit node in Inspect.

**Comment by Tanya-Solyanik** [comment]
I think the edit node should be displayed in the hierarchy, but it should contain the text (item1)

**Comment by John-Qiao** [comment]
Verified on .NET 7.0 test pass build: .NET 7.0.100-rc.1.22425.12, issue was fixed as below:
![4810-TestResult](https://user-images.githubusercontent.com/45864985/187142060-c8caff79-d12c-4201-b647-50bb8fb8696a.png)

**Comment by Amy-Li03** [comment]
Verified on .NET 8.0 latest build from main branch: NET 8.0.100-alpha.1.22451.15, issue was fixed, test result is same as above screenshot.
